CMakeLists.txt

```diff
@@ -58,15 +58,6 @@ set(CMAKE_EXPORT_COMPILE_COMMANDS "ON")
 set(BUILD_DOXYGEN FALSE CACHE BOOL "Build doxygen documentation on every make")
 set(ENABLE_MULTIVERSION_PROTOCOL_TEST FALSE CACHE BOOL "Enable nodeos multiversion protocol test")
 
-# add defaults for openssl
-if(APPLE AND UNIX AND "${OPENSSL_ROOT_DIR}" STREQUAL "")
-   if(CMAKE_SYSTEM_PROCESSOR STREQUAL "arm64")
-      set(OPENSSL_ROOT_DIR "/opt/homebrew/opt/openssl@3;/opt/homebrew/opt/openssl@1.1")
-   else()
-      set(OPENSSL_ROOT_DIR "/usr/local/opt/openssl@3;/usr/local/opt/openssl@1.1")
-   endif()
-endif()
-
```

Comment:
this is still required for `OpenSSL::Crypto` support

Reply:
Weird, I was able to build and run the tests without it! Do you know why?

Reply:
in `libfc/CMakeLists.txt`, there is `find_package(OpenSSL REQUIRED)`. Maybe this is enough?

Reply:
on macos?

Reply:
Also I cannot find any reference to `OPENSSL_ROOT_DIR` in any CMakeLists.txt.

Reply:
`OPENSSL_ROOT_DIR` is an input to cmake's openssl package
https://cmake.org/cmake/help/latest/module/FindOpenSSL.html#hints

It's here to help out on macos since homebrew doesn't install the openssl headers & libraries in a default searchable location because they conflict with the system provided (but not public api) libressl

Reply:
OK, I'll add these back, thanks @spoonincode!

Reply:
> on macos?

no, on ubuntu 22.04